usr/etc/flatpak/user/install

```diff
@@ -1,6 +1,5 @@
 org.mozilla.firefox
 org.freedesktop.Platform.ffmpeg-full//22.08
-com.raggesilver.BlackBox
```

Comment:
:+1: Removing this line will not remove the BlackBox flatpak from existing systems, but it will mean BlackBox will not be installed by default on new installs.  
This is good behaviour.